awscli/topics/config-vars.rst

```diff
@@ -476,6 +478,38 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+AWS CLI uses pager for output data that don't fit on the screen.
+
+By default if on Linux/MacOS, use ``less`` as the pager on Windows
+by default use ``more``
+
+    You can override the default pager with the following configuration
+    options. These are in order of precedence:
+    ``AWS_PAGER`` environment variable
+    ``cli_pager`` shared config variable
+    ``PAGER`` environment variable
+
+If you set any of the configuration options to an empty string
+(e.g. ``AWS_PAGER=""``) or use ``--no-cli-pager`` option in the command line the
+CLI will not send the output to a pager.
+
+    If ``LESS`` environment variable isn't set CLI set it to ``FRX`` (see "less" manual
```

Comment:
Similar to a previous comment, instead of using block quotes, let's make this a new subsubsection `Pager settings`